.ci/install_runtime.sh

```diff
@@ -65,7 +65,10 @@ if [ "$KATA_HYPERVISOR" = "nemu" ]; then
 fi
 
 if [ "$KATA_HYPERVISOR" = "qemu" ] && [ "$experimental_qemu" == "true" ]; then
+	runtime_repo="github.com/kata-containers/runtime"
+
 	echo "Enable qemu virtiofs configuration.toml"
+	sudo mv "$GOPATH/src/${runtime_repo}/cli/config/configuration-qemu-virtiofs.toml" "${PKGDEFAULTSDIR}"
 	sudo mv "${PKGDEFAULTSDIR}/configuration-qemu-virtiofs.toml" "${PKGDEFAULTSDIR}/configuration.toml"
```

Comment:
well, now this is odd, I think you can remove this step, what about move `configuration-qemu-virtiofs.toml` directly to `"${PKGDEFAULTSDIR}/configuration.toml"` ?

```sh
sudo mv "$GOPATH/src/${runtime_repo}/cli/config/configuration-qemu-virtiofs.toml" "${PKGDEFAULTSDIR}/configuration.toml"
```

Reply:
@devimc because when we do the `kata-runtime kata-check` this will be failing as it complains that `configuration-qemu-virtiofs.toml` is missing at ${PGDEFAULTSDIR}